src/runtime/callbacks.js

```diff
@@ -172,10 +175,21 @@ export class Callbacks {
    * @return {boolean} Whether the callback has been found.
    */
   triggerPaymentResponse(responsePromise) {
-    return this.trigger_(
-      CallbackId.PAYMENT_RESPONSE,
-      responsePromise.then(res => res.clone())
+    this.paymentResponsePromise_ = responsePromise.then(
+      res => {
+        this.trigger_(
+          CallbackId.PAYMENT_RESPONSE,
+          Promise.resolve(res.clone())
+        );
+      },
+      reason => {
+        if (isCancelError(reason)) {
+          return;
+        }
+        throw reason;
```

Comment:
I'm not sure on the sequence of events, but if this throws, won't pay-flow.js execution halt at pay-flow.js:201 resulting in no logging of any failure?

Reply:
It's an asynchronous error because it happens in the error part of a promise.  It was throwing these errors before also, I am just suppressing it if you close the popup.